recipes/minizip/all/conanfile.py

```diff
@@ -0,0 +1,138 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class MinizipConan(ConanFile):
+    name = "minizip"
+    description = "minizip is a zip manipulation library written in C "
+    "that is supported on Windows, macOS, and Linux."
+    topics = ("conan", "minizip", "compression")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/nmoinvaz/minizip"
+    license = "Zlib"
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake_find_package"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "compat": [True, False],
+        "zlib": [True, False],
+        "bzip2": [True, False],
+        "zstd": [True, False],
+        "lzma": [True, False],
+        "pkcrypt": [True, False],
+        "wzaes": [True, False],
+        "libcomp": [True, False],
+        "openssl": [True, False],
+        "libbsd": [True, False],
+        "brg": [True, False],
+        "signing": [True, False],
+        "compress_only": [True, False],
+        "decompress_only": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "compat": True,
+        "zlib": True,
+        "bzip2": True,
+        "lzma": True,
+        "zstd": True,
+        "pkcrypt": True,
+        "wzaes": True,
+        "libcomp": False,
+        "openssl": False,
+        "libbsd": True,
+        "brg": False,
+        "signing": True,
+        "compress_only": False,
+        "decompress_only": False
+    }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+    _cmake = None
```

Comment:
```suggestion
    _cmake = None
    
    @property
    def _source_subfolder(self):
        return "source_subfolder"

    @property
    def _build_subfolder(self):
        return "build_subfolder"
```

Reply:
Applied in fc2ffd1c0748cad03326e10aa55df3d4f289a046